src/signing/networkMessages/polkadot-testnet.js

```diff
@@ -42,6 +42,29 @@ export async function StakeTx(senderAddress, { to, amount }, network) {
   return await getSignMessage(senderAddress, transactions)
 }
 
+export async function ClaimRewardsTx(senderAddress) {
+  let allClaimingTxs = []
+  const api = await getAPI()
+  const stakerRewards = await api.derive.staking.stakerRewards(senderAddress)
```

Comment:
still here

Reply:
Yep, but this is the derive call to get the rewards info.

Reply:
we agreed to move towards using our API query for rewards in another PR